From: Christian Lamparter <chunkeey@gmail.com>
Date: Thu, 7 Jun 2018 19:29:12 +0200
Subject: bcm53xx: add LED status label alias for Meraki MR32

add an led-status alias label. This is used by OpenWrt's LED
DTS lookup function to identifiy the indicator LED

Signed-off-by: Christian Lamparter <chunkeey@gmail.com>

--- a/arch/arm/boot/dts/bcm53016-meraki-mr32.dts
+++ b/arch/arm/boot/dts/bcm53016-meraki-mr32.dts
@@ -27,6 +27,7 @@
 
 	aliases {
 		serial1 = &uart2;
+		led-status = &led_status;
 	};
 
 	leds {
@@ -68,7 +69,7 @@
 			max-brightness = <255>;
 		};
 
-		green {
+		led_status: green {
 			/* SYS-LED 1 - Tricolor */
 			function = LED_FUNCTION_POWER;
 			color = <LED_COLOR_ID_GREEN>;
